pkg/expression/builtin_compare.go

```diff
@@ -66,6 +66,7 @@ var (
 	_ builtinFunc = &builtinLeastVectorFloat32Sig{}
 	_ builtinFunc = &builtinIntervalIntSig{}
 	_ builtinFunc = &builtinIntervalRealSig{}
+	_ builtinFunc = &builtinLeastVectorFloat32Sig{}
```

Comment:
Mistake?

Reply:
This code should be included in pre PR. I don't know how it get here.
Anyway, I remove it.